### ISSUE TITLE
Grab all viewmodel data in Async rule without creating dependencies

### ISSUE THREAD

**Issue by jstrandelid** [Knockout-Contrib/Knockout-Validation#559]
I'm in a bit of trouble and need som help.

I've created an Async rule for validation on the server. 
The rule is applied to one observable, in the viewmodel, but the validation on the server requires all data in viewmodel. 
As far as I understand, this will make all observables in the viewmodel triggers to this rule due to the fact that the rule is grabbing all data in the viewmodel with ko.toJS.

In [this JSFiddle example](http://jsfiddle.net/bcy5zvm1/1/), the Data1 observable is extended with a server validation. The server validation requires all data in the viewmodel, but only the Data1 field should act as an trigger. However when I change Data2 (or Data3) the servervalidation of Data1 is triggered.

Is it possible grab all data from the viewmodel without creating dependencies?

**Comment by crissdev** [comment]
@jstrandelid You might want use [peek](http://knockoutjs.com/documentation/computed-dependency-tracking.html#controlling-dependencies-using-peek) to retrieve the value of each observable in the view model if you want to avoid dependencies being created.

```js
// Instead of calling ko.toJS, grab each value individually
// var json = ko.toJS(param.model());

var model = param.model();

var json = {
   data1: model.param1.peek(),
   data2: model.param1.peek(),
   data3: model.param1.peek()
};
```

---
Closing this issue as it's not an issue related to the library.
